### PR TITLE
CCD-1026 CCD / ExUI: Integrate "Default State" Workbasket Input Filter with CaseRoles Permissions

### PR DESCRIPTION
**Pull request by padmapriyanalam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-1026

### Change description ###
CCD / ExUI: Integrate "Default State" Workbasket Input Filters with CaseRoles Permissions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/ccd/endpoint/ui/UserProfileEndpointTest.java [hunk 1]
```diff
@@ -53,9 +53,9 @@ public void validUser() throws Exception {
         assertNull(userProfile.getChannels());
 
         final JurisdictionDisplayProperties[] jurisdictions = userProfile.getJurisdictions();
-        assertEquals(3, jurisdictions.length);
+        assertEquals(4, jurisdictions.length);
```
Comment (danlysiak) [inline src/test/java/uk/gov/hmcts/ccd/endpoint/ui/UserProfileEndpointTest.java:56]: Let's add an assertion for the first element too then

File: src/test/java/uk/gov/hmcts/ccd/v2/internal/controller/UIDefinitionControllerIT.java [hunk 2]
```diff
@@ -89,4 +91,34 @@ public void shouldReturnWorkSearchInputDefinitions() throws Exception {
...
+
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+        scripts = {
+            "classpath:sql/insert_cases.sql", "classpath:sql/insert_case_users.sql"})
+    public void shouldReturnWorkBasketInputFilterWithCaseRoleStates() throws Exception {
+        final MvcResult result =
+            mockMvc.perform(get("/internal/jurisdictions")
+                .contentType(MediaType.APPLICATION_JSON)
+                .param("access", "create")
```
Comment (danlysiak) [inline src/test/java/uk/gov/hmcts/ccd/v2/internal/controller/UIDefinitionControllerIT.java:103]:
```suggestion
                .param("access", "read")
```

File: src/test/java/uk/gov/hmcts/ccd/endpoint/ui/UserProfileEndpointTest.java (hunk 1 above)
Comment (padmapriyanalam) [reply src/test/java/uk/gov/hmcts/ccd/endpoint/ui/UserProfileEndpointTest.java:56]: added the assertion for the first element

**Review by danlysiak — APPROVED** [review]
👍

**Review by RebeccaBaker — APPROVED** [review]

**Pull request merged**